### PR TITLE
Add reed-helgemo recommendations

### LINKED ISSUE #5: reed-helgemo checklist
- [x] Create a branch
- [x] Add a file
- [ ] Make a PR
- [ ] Merge

### PR DESCRIPTION
**Pull request by reed-helgemo**
Adding Twin Cities recommendations

Closes #5

### CODE REVIEW

File: reed-helgemo-rec.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Twin Cities
+
+### Places to Eat
+- Town Hall Brewery
+- Surly Brewery
+- Great Water's Brewery
+- Ruam Mit Thai
```
Comment (brian-turner) [inline reed-helgemo-rec.md:7]: @reed-helgemo Where is Ruam Mit Thai?  :thailand:

File: reed-helgemo-rec.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### Twin Cities
+
+### Places to Eat
+- Town Hall Brewery
+- Surly Brewery
+- Great Water's Brewery
```
Comment (brian-turner) [inline reed-helgemo-rec.md:6]: Are these all places to eat or more geared to getting a :beer:?

**Review by brian-turner — COMMENTED** [review]
See comments in review.

File: reed-helgemo-rec.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Twin Cities
+
+## Places to Eat  (and drink)
```
Comment (reed-helgemo) [inline reed-helgemo-rec.md:3]: Hi Brian - I updated my recommendations document to reflect that these are places to both eat delicious food and consume delicious craft beer!

**Review by brian-turner — APPROVED** [review]
Approving your changes.

File: reed-helgemo-rec.md (hunk 1 above)
Comment (reed-helgemo) [reply reed-helgemo-rec.md:7]: Ruam Mit is walking distance from Securian.  Not too far from Wells Fargo.

**Pull request merged**